.gitignore

```diff
@@ -1,2 +1,10 @@
 crane_darwin_amd64
 crane_linux_amd64
+
+# IntelliJ IDE files
+.idea/
+atlassian-ide-plugin.xml
+*.iml
+
+# Cloud9 IDE files
+.c9/
```

Comment:
I'm not a huge fan of putting all the different project files into the `.gitignore` file. My suggestion would be to remove them here and put them in your global `.gitignore`.